### PR TITLE
Feature3

### PR DESCRIPTION
**Pull request by gparsloe**
Hey. Testing pull request.

### CODE REVIEW

**Comment by AbbyG88** [comment]
Prior to merging, let's spruce it up a bit.

File: hello.txt [hunk 1]
```diff
@@ -1,3 +1,60 @@
 hello this is a message from brackets...
 
-Ok. I checked out feature3 branch before writing this. Are we really only on 3?
\ No newline at end of file
+Ok. I checked out feature3 branch before writing this. Are we really only on 3?
+
+Sprucing it up with a few Hamilton lyrics:
+
+ELIZA:
+Un deux trois quatre
```
Comment (gparsloe) [inline hello.txt:8]: Can you translate this to some other languages?

File: hello.txt [hunk 2]
```diff
@@ -0,0 +1,60 @@
+hello this is a message from brackets...
+
+Ok. I checked out feature3 branch before writing this. Are we really only on 3?
+
+Sprucing it up with a few Hamilton lyrics:
+
+ELIZA:
+Un deux trois quatre
```
Comment (AbbyG88) [inline hello.txt:8]: What's up with the French?

**Review by AbbyG88 — COMMENTED** [review]
Looks good. Just wondering why you didn't stick to English.

File: hello.txt (hunk 1 above)
Comment (gparsloe) [reply hello.txt:8]: Ha ha ha... Why don't you write to Lin Manuel Miranda and ask him for the translation!?

File: hello.txt (hunk 2 above)
Comment (gparsloe) [reply hello.txt:8]: Very funny! I think you can handle a few numbers. If not, just enjoy the rest. : )

**Pull request merged**